src/scss/_layout.scss

```diff
@@ -0,0 +1,50 @@
+.alignleft {
+  float: left;
+}
+
+.alignright {
+  float: right;
+}
+
+a img {
```

Comment:
Sorry, I was referring to adding these rules to the `<a>` tag itself, not the inner `<img>`, so that images in figures would be centered at the -xs breakpoint.  `text-align` won't do much applied to the `<img>` itself.  If you want to center the images at the -xs breakpoint, I'd suggest changing this selector from `a img` to `.figure a`; otherwise, if you're good with the images being left-aligned at -xs, you could just remove this rule entirely.  I think that, with caption text being left-aligned currently, it might look best without the image centering, but it's your call.

Reply:
Unless I'm missing something, this can probably be removed entirely at this point since xs- the images are @100% anyway, so there's not going to be any aligning either way.